### PR TITLE
Add send inspection path after upload

### LINKED ISSUE #430: Send inspection url as telemetry
**Describe the new feature you would like to see**
When the uploader has uploaded inspection data, the URL should be sent as telemetry.

**Describe the solution you'd like**

**How will this feature affect the current Threat Model?**
N/A

### PR DESCRIPTION
**Pull request by monkeyysups**
closes #430 
From Slim upload, it sends image guid.
From Blob upload, it sends etag.
From local it sends the file path. 
The paths is sent with "TOPIC_ISAR_INSPECTION_RESULT"

### CODE REVIEW

File: src/isar/storage/blob_storage.py [hunk 1]
```diff
@@ -40,13 +40,14 @@ def store(self, inspection: Inspection, metadata: MissionMetadata):
             inspection=inspection, metadata=metadata, filename=local_path.name
         )
 
-        self._upload_file(path=local_path, data=inspection.data)
+        inspection_path = self._upload_file(path=local_path, data=inspection.data)
         self._upload_file(path=local_metadata_path, data=metadata_bytes)
+        return inspection_path
```
Comment (Christdej) [inline src/isar/storage/blob_storage.py:45]: Double `self._upload_file`, only one needed
```suggestion
        return self._upload_file(path=local_path, data=inspection.data)
```

File: src/isar/storage/blob_storage.py [hunk 2]
```diff
@@ -40,13 +40,14 @@ def store(self, inspection: Inspection, metadata: MissionMetadata):
...
 
-    def _upload_file(self, path: Path, data: bytes):
+    def _upload_file(self, path: Path, data: bytes) -> str:
         blob_client = self._get_blob_client(path)
         try:
-            blob_client.upload_blob(data=data)
+            inspection_path = blob_client.upload_blob(data=data)
```
Comment (Christdej) [inline src/isar/storage/blob_storage.py:50]:
```suggestion
            blob_properties = blob_client.upload_blob(data=data)
```
As it is not an inspection path that is returned from `upload_blob`

File: src/isar/storage/uploader.py [hunk 3]
```diff
@@ -118,10 +121,37 @@ def _upload(self, upload_item: UploaderQueueItem) -> None:
...
+            {
+                "isar_id": settings.ISAR_ID,
+                "robot_name": settings.ROBOT_NAME,
+                "step_id": inspection.id,
+                "inspection_path": inspection_path,
+                "timestamp": datetime.utcnow(),
+            },
+            cls=EnhancedJSONEncoder,
+        )
+        print(inspection_path)
```
Comment (Christdej) [inline src/isar/storage/uploader.py:152]: remove print

**Review by Christdej — CHANGES_REQUESTED** [review]

File: src/isar/storage/blob_storage.py (hunk 1 above)
Comment (monkeyysups) [reply src/isar/storage/blob_storage.py:45]: done

File: src/isar/storage/blob_storage.py (hunk 2 above)
Comment (monkeyysups) [reply src/isar/storage/blob_storage.py:50]: done

File: src/isar/storage/uploader.py (hunk 3 above)
Comment (monkeyysups) [reply src/isar/storage/uploader.py:152]: done

**Review by Christdej — APPROVED** [review]

**Pull request merged**